standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetastoreDirectSqlUtils.java

```diff
@@ -609,4 +609,16 @@ public static void throwMetaOrRuntimeException(Exception e) throws MetaException
       throw new RuntimeException(e);
     }
   }
+
+  static int getCountOfQuery(PersistenceManager pm, String queryText, Object[] params) {
```

Comment:
nit: how about declaring this method as `getCountOfQuery(Query query, Object[] params)`

Reply:
Done.